static/src/js/constants/collectionSortKeys.js

```diff
@@ -1,6 +1,6 @@
 export const collectionSortKeys = {
   scoreDescending: '-score',
   usageDescending: '-usage_score',
-  startDateAscending: 'start-date',
+  startDateAscending: 'start_date',
```

Comment:
this just fixes that issue we saw where start_date was not valid in the preferences in prod as a default sort method right? If so I won't write a ticket for it

Reply:
Yes, we discussed this a few days ago.

Reply:
Cool thanks less jira tickets for me to write